syntax = "proto2";

package protobufgenerator;

message SampleMessage{
    required string file_name = 1;
    required string file_content = 255;
}